test_reflectable/pubspec.yaml

```diff
@@ -11,10 +11,14 @@ environment:
 dependencies:
   build_runner: any
   glob: any
-  reflectable:
-    path: ../reflectable
+  reflectable: any
 dev_dependencies:
   analyzer: any
   build_test: any
   lints: any
+  reflectable_builder:
+    path: ../reflectable_builder
   test: ^1.25.0
+dependency_overrides:
```

Comment:
Instead of this you could use a pub workspace to develop the two packages.

Reply:
OK, using a pub workspace now, PTAL.